### PR TITLE
Avatar

### PR DESCRIPTION
**Pull request by malaykhakhar**
Created avatar component

### CODE REVIEW

File: components/Avatar/avatar.css [hunk 1]
```diff
@@ -0,0 +1,38 @@
+@import url("/style.css");
+
+.container-avatars {
```
Comment (yaswanthmyneni) [inline components/Avatar/avatar.css]: you can remove and use common class for every container as said in alert pr.

File: components/Avatar/avatar.css [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+.container-avatars {
+  width: 40rem;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.avatar {
+  /* background-color: ; */
```
Comment (yaswanthmyneni) [inline components/Avatar/avatar.css]: remove comments.

File: components/Avatar/avatar.css (hunk 1 above)
Comment (malaykhakhar) [reply components/Avatar/avatar.css]: Done

File: components/Avatar/avatar.html [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+          <img src="/components/images/boy-avatar.svg" alt="avatar" class="avatar avatar-m" />
+          <img src="/components/images/girl-avatar.svg" alt="avatar" class="avatar avatar-s" />
+          <img src="/components/images/boy-avatar.svg" alt="avatar" class="avatar avatar-xs" />
+        </div>
+
+        <iframe
+  src="https://carbon.now.sh/embed/Yxfeeibl1B7tqNbpO3fw"
+  style="width: 680px; height: 409px; border:0; transform: scale(1); overflow:hidden;"
+  sandbox="allow-scripts allow-same-origin">
+</iframe>
```
Comment (sri-rishi) [inline components/Avatar/avatar.html]: Give tab spacing to look code good

**Review by sri-rishi — CHANGES_REQUESTED** [review]
Except than this looks good to merge

**Pull request merged**